pkg/secrets/manager.go

```diff
@@ -23,13 +23,13 @@ import (
 
 // Manager provides the interface for providing stack encryption.
 type Manager interface {
```

Comment:
From what I can tell, the `Manager` type isn't ever exposed to users of the SDK, which means this API change is backward compatible. If I'm wrong about that assumption, please let me know :) 
Since `json.RawMessage` is more narrow than `interface{}`, I suppose we'd be OK either way.